desktop/dev-environments.md

```diff
@@ -55,6 +55,38 @@ The simplest way to get started with Dev Environments is to create a new environ
 >
 > When cloning a Git repository using SSH, ensure you've added your SSH key to the ssh-agent. To do this, open a terminal and run `ssh-add <path to your private ssh key>`.
 
+> **Note**
+>
+> If you have enabled the WSL2 integration in Docker Desktop for Windows, make sure you have an SSH agent running in your WSL2 distribution. Otherwise, click on the expansible section below to learn how to start a new SSH agent in case there is not one running already.
+The expansible section will contain a piece of script that you shall append to the bottom of your `~/.profile`. Then, re-open a new WSL2 terminal afterwards.
```

Comment:
instead of mentioning only `~/.profile` file, can we mention instead:
> your profile file (ie : `~/.profile`, `~/zshrc`, ...)`
And isn't a `source` command enough? do we really need to re-open a terminal?

Reply:
Added more profile files, thanks